chained/proxy.go

```diff
@@ -71,18 +71,19 @@ type nopCloser struct{}
 
 func (c nopCloser) close() {}
 
-// CreateDialers creates a list of Proxies (balancer.Dialer) with supplied server info.
-func CreateDialers(configDir string, proxies map[string]*config.ProxyConfig, uc common.UserConfig) []balancer.Dialer {
+// CreateDialers creates a list of Proxies (orchestrator.Dialer) with supplied server info.
```

Comment:
there's inconsistency in naming. there are several places referencing 'orchestrator' while others just name 'bandit'